contracts/VaultV3.vy

```diff
@@ -443,6 +459,9 @@ def _convert_to_assets(shares: uint256, rounding: Rounding) -> uint256:
     """ 
     assets = shares * (total_assets / total_supply) --- (== price_per_share * shares)
     """
+    if shares == MAX_UINT256:
```

Comment:
Should we also special case 0?

Reply:
Well people shouldnt be sending 0 in any of the external functions as an input.

But there is the case that the report event calls convert_to_assets twice with a 0 value if no fees are charged. So it would save gas there

Reply:
added https://github.com/yearn/yearn-vaults-v3/pull/179/commits/ecf23db5fbb33f8c7afd4c62d1b331483262a80e